pinot-udf/src/main/java/org/apache/pinot/scalar/function/hypertrace/HypertraceScalarFunctions.java

```diff
@@ -24,6 +28,17 @@ public static String defaultString(String value, String defaultValue) {
             replaceNullStringWithNull(value), replaceNullStringWithNull(defaultValue)));
   }
 
+  @ScalarFunction(name = STRING_EQUALS_FUNCTION_NAME)
+  public static String stringEquals(String s1, String s2) {
```

Comment:
According to https://docs.pinot.apache.org/users/user-guide-query/scalar-functions#adding-user-defined-scalar-functions inputs and outputs for scalar functions can only be Integer, Long, Double or String

Reply:
Hmm this is unfortunate. Let's see how this approach works.

Reply:
Why can't we do `colA = colB` in the query?

Reply:
Because you cannot do that on a selection.